cypress/integration/Authentication/emailCheck_spec.js

```diff
@@ -0,0 +1,33 @@
+const token = "token";
+const userDetails = {
+    email: "maggie@summit.com",
+    first_name: "Maggie",
+    id: 1,
+    is_staff: true,
+    last_name: "Huang",
+};
+const fakeDetails = {
+    email: "iamnotreal@summit.com",
+    first_name: "fake",
+    id: 2,
+    is_staff: true,
+    last_name: "person",
+};
+
+describe("emailCheck_spec", () => {
+    it("loads email check page", () => {
+        cy.visit("");
+        cy.get("[data-cy=nextButton]");
```

Comment:
this should contain an assertion to check against the button, not just grabbing the button data.

```
cy.get("[data-cy=nextButton]").should("contain", "SIGN IN");
```